trick_source/trick_utils/comm/test/TCConnectTest.cpp

```diff
@@ -20,7 +20,7 @@ class TCConnectTest : public testing::Test {
          /* device */
          device = (TCDevice *) malloc(sizeof(TCDevice));
          memset( (void *)device,'\0',sizeof(TCDevice) );
```

Comment:
Another opportunity to fix someone else's code! The argument to `sizeof` should be the [variable itself](https://softwareengineering.stackexchange.com/questions/201104/sizeof-style-sizeoftype-or-sizeof-variable), when possible, not its type. That way, if the type changes in the future, this line doesn't need to be changed.

Reply:
Okey poke. Will fix.